docs/repositories-configure/code-patterns.md

```diff
@@ -22,6 +22,11 @@ To make it easier to find relevant patterns, you can use the sidebar to filter t
 
 ![Filtering code patterns](images/code-pattern-category.png)
 
+!!! tip
+    To enable all patterns inside a Category, you can use the filter to list all patterns inside that Category and then use the **Enable all** button.
+    
+    As an example, if you want to enable **all** Security patterns of a tool, instead of filtering and then enabling all the patterns one by one, you can simply add the filter and then click on the **Enable all** button.
```

Comment:
Same suggestion as above, and removed the adverb "simply" because it's good practice to omit adverbs in technical documentation:

* https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/simply
* https://docs.microsoft.com/en-us/style-guide/word-choice/use-simple-words-concise-sentences

```suggestion
    As an example, if you want to enable **all** Security patterns of a tool, instead of filtering and then enabling all the patterns one by one, you can add the filter and click **Enable all**.
```

Reply:
Really nice suggestion! Didn't know about this good practice 🙏

Reply:
Yes, if you're curious you can read a more thorough explanation about it on Google's style guide :nerd_face: 

https://developers.google.com/tech-writing/one/clear-sentences#minimize_certain_adjectives_and_adverbs_optional